packages/hugo.rb

```diff
@@ -14,8 +14,8 @@ class Hugo < Package
     source_url 'https://github.com/gohugoio/hugo/releases/download/v0.72.0/hugo_0.72.0_Linux-32bit.tar.gz'
     source_sha256 '573e41bf48b678ed909d4cf5b053fb4f259d672c0df7118ec980dcba000349fd'
   when 'x86_64'
-    source_url 'https://github.com/gohugoio/hugo/releases/download/v0.72.0/hugo_0.72.0_Linux-64bit.tar.gz'
-    source_sha256 '28353611210d48c681f1f83a64ce36972d010e751f2794122db80f5060fe933d'
+    source_url 'https://github.com/gohugoio/hugo/releases/download/v0.72.0/hugo_extended_0.72.0_Linux-64bit.tar.gz'
+    source_sha256 '11c46ecc3ad6d96f94515358e5528d01dfe88f4956b56027c85e9a516348ef12'
```

Comment:
I wonder if they other architectures need this also?  Maybe change for all?

Reply:
Just checked the releases page at https://github.com/gohugoio/hugo/releases.  Other architectures don't have this feature.

Reply:
> I wonder if the other architectures need this also? Maybe change for all?

Lol : check the initial description of the PR : https://github.com/skycocker/chromebrew/pull/4135#issue-436692957

Reply:
I only focus in on the code. LoL